mobile_app_connector/models/app_tile.py

```diff
@@ -105,7 +105,8 @@ def render_tile(self, tile_data=None):
                         tile_json.update(tile._render_single_tile(record))
                         res.append(tile_json.copy())
             else:
-                res.append(tile_json)
+                if tile_json['Type'] != 'Letter':
```

Comment:
What is this exclusion used for? You should add a comment explaining because reading the code it's confusing.